plugin_playback_guiplayer/__init__.py

```diff
@@ -100,12 +124,19 @@ def _play(self, message):
                 self.bus.emit(Message("playback.display.audio.type"))
 
         time.sleep(0.5)
-        self.bus.emit(
-            Message(
-                "gui.player.media.service.play", {
-                    "track": track, "mime": mime, "repeat": repeat}))
-        LOG.debug('Player Emitted gui.player.media.service.play')
-        self.send_meta_to_player()
+        if self.web_player:
+            self.bus.emit(Message("gui.web.media.service.play",
```

Comment:
The handler for this doesn't exist yet, is that right? 
I haven't been able to find where what would actually play the media

Reply:
Is this still an issue at all? Or can this be merged?

Reply:
The webpage itself plays the media.. any media loaded by a webpage in a browser.. will be handled by the browser in this case (QtWebEngine/Chromium)

Reply:
The only thing that is being passed here is the URL of the page to load and display in the browser

Reply:
Yeah right, but where is the handler for `playback.display.web.video.type` to load the url provided?

I couldn't see it in mycroft-gui or this plugin or the playback-control-skill - so it just seems like we're missing a piece of the puzzle?